src/locales/en-GB/translation.json

```diff
@@ -0,0 +1,56 @@
+{
```

Comment:
The EN keys were defined but empty in the task linked to this PR. Intuitively I then created a locale file for en-GB that will probably be populated later on (also allowing us to test intl solutions in the meantime). But I guess we can delete it for now if you prefer, since it's not used at all that makes sense for sure